lib/init/rc.shutdown

```diff
@@ -16,20 +16,18 @@ main() {
         dd count=1 bs=512 if=/dev/random of=/var/random.seed
     }
 
-    halt -w
-
     log "Sending TERM signal to all processes..."; {
-        killall5 -TERM
+        killall5 TERM
         sleep 1
     }
 
     log "Sending KILL signal to all processes..."; {
-        killall5 -KILL
+        killall5 KILL
```

Comment:
Does `killall5` work without `-` in `busybox` as well. Might need to investigate here.

Reply:
I'm not sure. You  try it. I'm busy. *lol*

Reply:
-s done